attestation/commandrun/tracing_linux.go

```diff
@@ -292,18 +536,47 @@ func getPPIDFromStatus(status []byte) (int, error) {
 	return 0, nil
 }
 
-func getSpecBypassIsVulnFromStatus(status []byte) bool {
-	statusStr := string(status)
-	lines := strings.Split(statusStr, "\n")
-	for _, line := range lines {
-		if strings.Contains(line, "Speculation_Store_Bypass:") {
-			parts := strings.Split(line, ":")
-			isVuln := strings.TrimSpace(parts[1])
-			if strings.Contains(isVuln, "vulnerable") {
-				return true
-			}
+func (p *ptraceContext) readSyscallData(pid int, addr uintptr, size int) ([]byte, error) {
+	buf := make([]byte, size)
+	n := 0
+	for n < size {
+		word := make([]byte, WORD_SIZE)
+		_, err := unix.PtracePeekData(pid, uintptr(addr)+uintptr(n), word)
```

Comment:
ptracepeekdata is very slow -- that's why we use ProcessVMReadv when reading process data elsewhere.  is that possible here?